examples/emberjs/js/controllers/todo_controller.js

```diff
@@ -33,23 +34,25 @@
 				// Re-set our newly edited title to persist its trimmed version
 				this.set('bufferedTitle', bufferedTitle);
 				this.set('isEditing', false);
+                return false;
 			},
 
 			cancelEditing: function () {
 				this.set('bufferedTitle', this.get('title'));
 				this.set('isEditing', false);
+                return false;
 			},
 
 			removeTodo: function () {
 				this.removeTodo();
+                return false;
```

Comment:
@sindresorhus @phillipsj returning false from an action prevents any parent controllers from handling the action. By default, I leave that out of my actions, and then add it in if necessary. It should be unnecessary here, but I haven't run this code yet